src/modules/departments/ui/Departments.page.tsx

```diff
@@ -0,0 +1,36 @@
+import { useDepartments } from "../mutation/departments";
+import { useState } from "react";
+import { Department } from "cv-graphql";
+import PageTemplate from "@/components/pageTemplate/PageTemplate";
+import { Action } from "@/components/pageTemplate/ActionsMenu";
```

Comment:
Do not forget about correct naming, we name all folders and filed in kebab-case like "cv-graphql" here

Reply:
done